src/robot/parsing/populators.py

```diff
@@ -145,7 +145,7 @@ def _populate_children(self, datadir, children, include_extensions,
 
     def _get_include_suites(self, path, incl_suites):
         if not isinstance(incl_suites, SuiteNamePatterns):
-            incl_suites = SuiteNamePatterns(self._create_included_suites(incl_suites))
+            incl_suites = SuiteNamePatterns(incl_suites)
```

Comment:
If `_create_included_suites` isn't used anymore, the helper method itself should also be removed.